scripts/py_matter_idl/matter/idl/tests/inputs/large_lighting_app.matter

```diff
@@ -1917,7 +1917,7 @@ endpoint 0 {
     callback attribute packetUnicastTxCount;
     callback attribute currentMaxRate;
     callback attribute overrunCount;
-    ram      attribute featureMap default = 3;
+    ram      attribute featureMap default = 0;
```

Comment:
As above, and for the other case in this file.

Reply:
Same as above - I had to fix invalid maps with non-existent data.